src/pages/index.js

```diff
@@ -61,7 +61,7 @@ function Cards() {
       label: translate({ message: 'Cross Platform' }),
       description: translate({
         message:
-          'Bundle binaries for all major desktop platforms (mobile & WASM coming soon).',
+          'Bundle binaries for all major desktop platforms (mobile coming soon).',
```

Comment:
Since this is a fairly high-visibility page, do we have a link to an issue or discord discussion we could maybe just throw as a comment on this PR for why we're removing it?

Reply:
No and i didn't want to make a fuss about it to not cause any panics, because actually _nothing_ changes. We can't create tauri apps (with webviews) that run in wasm apps anyway, and many people interpreted this as "wasm support in the browser" which was always supported.
Also it was removed once before already: https://github.com/tauri-apps/tauri-docs/pull/168 and re-added (accidentally?) here https://github.com/tauri-apps/tauri-docs/pull/337